crates/net/discv5/src/lib.rs

```diff
@@ -197,14 +197,14 @@ impl Discv5 {
         //
         // 3. add boot nodes
         //
-        Self::bootstrap(bootstrap_nodes, &discv5).await?;
+        bootstrap(bootstrap_nodes, &discv5).await?;
 
         let metrics = Discv5Metrics::default();
 
         //
         // 4. start bg kbuckets maintenance
         //
-        Self::spawn_populate_kbuckets_bg(
+        spawn_populate_kbuckets_bg(
```

Comment:
can we make these functions of self?
I think we can also give the instance a clone of the bootstrap nodes

so this becomes just `self.bootstrap().await`

Reply:
functions of self = methods. not possible here, since the self type, in this case `Discv5`, hasn't been instantiated yet